lib/assets/Html.js

```diff
@@ -214,9 +213,25 @@ extendWithGettersAndSetters(Html.prototype, {
                     throw err;
                 }
             }
-            if (isEmpty) {
-                // Remove the sole text node caused by the text === '' hack above:
-                this._parseTree.removeChild(this._parseTree.firstChild);
+            if (isFragment) {
+                this._parseTree = document.body;
+                // Install an assortment of bound methods from the document object on the parse tree:
+                [
+                    'createElement', 'createElementNS',
+                    'createTextNode', 'createComment', 'createDocumentFragment', 'createProcessingInstruction', 'importNode', 'adoptNode',
+                    'createAttribute', 'createAttributeNS',
+                    'createEvent',
+                    'createTreeWalker',
+                    'createNodeIterator',
+                    'getElementById',
+                    'getElementsByName'
```

Comment:
Implemented in 764d844 (my intuition doesn't fail me, adding `head` actually broke a couple of tests, so I ended up skipping that :)